CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # master
 *Please put new entries at the top.
 
+1. Add `Lifetime.of(_:)` for retrieve the lifetime of AnyObject and AnyObject.reactive.lifetime is now available. (#3614, kudos to @ra1028)
```

Comment:
[Nit] Introduce `Lifetime.of(_:)` which retrieves the lifetime of any Objective-C or Swift native object.